### ISSUE TITLE
Password length — JSON parse error

### ISSUE THREAD

**Issue by vmatekole** [digitalbitbox/dbb-app#284]
When creating wallets with large length passwords e.g. 70+ with both alphanumeric and symbol-based characters, there appears to be a very serious issue in that passwords are not valid. With various permutations, I get the attached message JSON parse error. I haven't had time to figure exact lengths where issue starts but I do get consistent errors at 100+ characters.  

Thankfully, no funds were lost but it does appear that there are some restrictions concerning password length and/or character types that are not validated upon creating the wallet and only appear when trying to log back into the wallet, putting at risk the users funds.

OS: OSX 10.12.6

**Comment by vmatekole** [comment]
Additionally, the password generator used was Keepass X.

Example password:

9b^cqz9b;9pPW^`^)]/[R^Vx8:K-Y-Hq:f@a}BX-+AFDeN3m6Cc{*$pzjz{:p`f$X[wXQ]@XjBNP'/;Ju>H`2[BMxZNP_UhbE2yz?uyH7i(Cfmq'.}/;\{D,

**Comment by jonasschnelli** [comment]
Have you tried current master of the prebuilt binary from digitalbitbox.com?

There is a fix for the password max length in https://github.com/digitalbitbox/dbb-app/pull/274.
The next release which includes that fix will be release soon.

**Comment by vmatekole** [comment]
@jonasschnelli No, I just bought a bitbox and downloaded the latest binaries for the website. When will it be published?

**Comment by jonasschnelli** [comment]
This should be fixed now in the current available software.
Can you confirm @vmatekole?
